clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/GCEUtil.groovy

```diff
@@ -920,7 +920,7 @@ class GCEUtil {
       : []
   }
 
-  static ServiceAccount buildScheduling(BaseGoogleInstanceDescription description) {
+  static Scheduling buildScheduling(BaseGoogleInstanceDescription description) {
```

Comment:
How did this ever work before?

Reply:
In line: https://github.com/spinnaker/clouddriver/blob/master/clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/GCEUtil.groovy#L924 
It creates a Scheduling object in runtime. Groovy is flexible enough to allow a different object in runtime and it only pass the object to the InstanceProperties object: 
https://github.com/spinnaker/clouddriver/blob/master/clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/handlers/BasicGoogleDeployHandler.groovy#L390
https://github.com/spinnaker/clouddriver/blob/master/clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/handlers/BasicGoogleDeployHandler.groovy#L428

that is how this works but I don't like it. It's really hard to follow what is going on with groovy. That is why I moved it to java.

Reply:
OK.  I suppose in some future we'll change GCEUtil to java too.

Reply:
yes, that is the plan